solutions/src/main/java/ru/sberbank/school/task02/validators/FxConversionValidator.java

```diff
@@ -0,0 +1,30 @@
+package ru.sberbank.school.task02.validators;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+/**
+ * Валидирует аргументы
+ * Если аргумент равен null, то кидает FxConversionException
+ * Created by Gregory Melnikov at 11.05.2019
+ */
+public class FxConversionValidator {
+
+    public static void nullableClientOperation(ClientOperation operation) {
+        if (operation == null) {
+            throw new FxConversionException("ClientOperation must be non null");
+        }
+    }
+    public static void nullableSymbol(Symbol symbol) {
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-762574536>*